src/WingetCreateCore/Common/Serialization.cs

```diff
@@ -137,6 +137,18 @@ public static string ConvertYamlToJson(string yaml)
             return JsonConvert.SerializeObject(yamlObject);
         }
 
+        /// <summary>
+        /// Serializes the provided object and returns the serialized string.
+        /// </summary>
+        /// <typeparam name="T">Object type.</typeparam>
+        /// <param name="value">Object to be serialized.</param>
+        /// <returns>Serialized string.</returns>
+        public static string Serialize<T>(T value)
```

Comment:
No need for this to be generic, it can just take an object

Reply:
Changed parameter to take in an object